decidim-meetings/app/presenters/decidim/meetings/meeting_presenter.rb

```diff
@@ -149,7 +149,7 @@ def formatted_proposals_titles
       end
 
       def sanitized(content)
-        decidim_sanitize_editor(content)
+        decidim_sanitize_editor_admin(content)
```

Comment:
I had to double check, but as far as I see the `sanitized` method is only being used in line 85  by the `registration_email_custom_content` method. If that's the case then I think it'd be much better to just inline it so we don't allow more tags by mistake in other places in the future.

Basically this is what I have in mind:

```
      def registration_email_custom_content(links: false, all_locales: false)
        return unless meeting
        handle_locales(meeting.registration_email_custom_content, all_locales) do |content|
renderer = Decidim::ContentRenderers::HashtagRenderer.new(decidim_sanitize_editor_admin(content))
          renderer.render(links:).html_safe
        end
end
```

This was introduced in c1696f68d9c (just in case if you want to check it again)